content/cloud-docs/users-teams-organizations/permissions.mdx

```diff
@@ -100,11 +100,12 @@ Workspace admins have the following permissions on their workspace:
 - Read and write variables
 - Read and write state versions
 - Read and write workspace settings (workspace admins only) — this includes general settings, notification configurations, run triggers, and more.
-- Set or remove workspace permissions for any team (workspace admins only)
+- Set or remove workspace permissions for visible teams (workspace admins only)
+  - Workspace admins cannot view or manage teams that are are ["Secret"](/cloud-docs/users-teams-organizations/teams#team-visibility), unless they are also organization owners.
```

Comment:
Maybe this?  Also this is kind of related but also unrelated, isn't "workspace admins only" implied by the "Workspace admins have the following permissions" statement above all of these bullets? If so I'm wondering why we need it in some of these....

```suggestion
- Set or remove workspace permissions for visible teams (workspace admins only). Workspace admins cannot view or manage teams that are are ["Secret"](/cloud-docs/users-teams-organizations/teams#team-visibility), unless they are also organization owners.
```

Reply:
> Also this is kind of related but also unrelated, isn't "workspace admins only" implied by the "Workspace admins have the following permissions" statement above all of these bullets? If so I'm wondering why we need it in some of these....

There are a bunch of workspace permissions which can be [assigned to any team](https://www.terraform.io/cloud-docs/users-teams-organizations/permissions#general-workspace-permissions), but you need to be a workspace admin to do some things (like deleting the workspace, or changing the permissions of. other teams)

So I think it is useful to call those out ("hey, you need to be an admin to do these things")...maybe we could change the list to be things which **only** workspace admins do, as opposed to **all** the things workspace admins do? Something like "Workspace admins have all permissions described in [General Permissions](https://www.terraform.io/cloud-docs/users-teams-organizations/permissions#general-workspace-permissions), as well as the ability to..."?

Reply:
YESSSS I actually think I like that approach you just mentioned. Sorry to balloon this PR into more than you anticipated, but I would be interested in seeing how that looks :)

Reply:
Updated as described, to remove the duplication with the above section and focus the permissions here on the stuff exclusive to workspace admins. Let me know what you think @laurapacilio !
<img width="937" alt="Screen Shot 2022-05-19 at 3 14 09 PM" src="https://user-images.githubusercontent.com/10147422/169385049-54d3ef36-a528-488d-a578-8d25d2c0b0b2.png">